packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -452,7 +461,7 @@ class ScrollableBase extends Component {
 			ev.preventDefault();
 		}
 
-		this.animateOnFocus = true;
+		this.animateOnFocus = this.props.animate;
```

Comment:
There is no initial value of `animateOnFocus`. I guess it's safer to assign value, but leave it as-is for now.